### PR TITLE
refactor: script gen types template

### PR DESCRIPTION
**Pull request by LaamGinghong**
Update the template to make it more simpler.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/IduxFE/components/blob/main/docs/contributing.zh.md#commit)
- [x] Tests for the changes have been added/updated or not needed
- [x] Docs and demo have been added/updated or not needed

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Component style update
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Application (the showcase website) / infrastructure changes
- [ ] Other... Please describe:

### CODE REVIEW

**Comment by LaamGinghong** [comment]
@danranVm

File: packages/components/button/src/types.ts [hunk 1]
```diff
@@ -2,24 +2,28 @@ import type { DefineComponent } from 'vue'
...
+  ghost?: boolean
+  disabled?: boolean
+  loading?: boolean
+  size?: ComponentSize
+  shape?: ButtonShape
+  block?: boolean
+  icon?: string
 }
 
+export type ButtonProps = Readonly<ButtonOriginalProps>
```
Comment (danranVm) [inline packages/components/button/src/types.ts:17]: 这样会不会有跟 ReturnType 一样的问题？ 没那么直观。

Comment (danranVm) [reply packages/components/button/src/types.ts:17]: 已验证，没有问题。

**Comment by danranVm** [comment]
感谢 PR， LGTM.

**Review by danranVm — APPROVED** [review]

**Pull request merged**